source/server-discovery-and-monitoring/server-monitoring.rst

```diff
@@ -1104,7 +1104,9 @@ A monitor clears the connection pool when a server check fails with a network
 or command error (`Network or command error during server check`_).
 When the check fails with a network error it is likely that all connections
 to that server are also closed.
-(See `JAVA-1252 <https://jira.mongodb.org/browse/JAVA-1252>`_).
+(See `JAVA-1252 <https://jira.mongodb.org/browse/JAVA-1252>`_). When the check fails 
```

Comment:
done about first part

Reply:
Can you elaborate the second part a bit? Are you saying about adding a different connection closed reason? If so, I guess we already discussed it here: https://github.com/mongodb/specifications/pull/1170#discussion_r860087100
NOTE: at least for c# in this case it will be retryable exception

Reply:
I will think about it, but at first glance I doubt that it's possible to throw `PoolClearedError` for this case

Reply:
That's okay, I figured it wouldn't be possible in some drivers. The only strong requirement is that the error is retryable. It'd be preferable if the error communicated to the user how it happened though.

Reply:
done